ufo/chrome_policy.py

```diff
@@ -0,0 +1,73 @@
+"""The module for generating and outputing chrome policy."""
+
+from . import app, setup_required
+
+import flask
+import json
+
+import models
+
+
+def _MakeChromePolicyJson():
```

Comment:
nit: think this might now be better as lowercase with underscore to be consistent with everything else here, and moving forward to our agreement to use lower casing